src/azure-cli-core/azure/cli/core/commands/validators.py

```diff
@@ -119,3 +119,11 @@ def _dest_to_option(dest):
             forbidden_string = ', '.join(_dest_to_option(x) for x in included_forbidden)
             error = '{}\n\tnot applicable: {}'.format(error, forbidden_string)
         raise CLIError(error)
+
+
+def validate_edge_zone(cmd, namespace):  # pylint: disable=unused-argument
+    if namespace.edge_zone:
```

Comment:
`edge_zone` can be any string?

Reply:
Yes. The definition is a string. In practice, service surely does not allow all strings.